stock_barcodes_gs1_expiry/wizard/stock_barcodes_read_views.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>
+    <record id="view_stock_barcodes_read_packaging_form" model="ir.ui.view">
+        <field name="model">wiz.stock.barcodes.read</field>
+        <field name="inherit_id" ref="stock_barcodes.view_stock_barcodes_read_form" />
+        <field name="arch" type="xml">
+            <field name="lot_name" position="after">
+                <field name="use_date" readonly="1" force_save="1" invisible="1" />
+                <field
+                    name="expiration_date"
+                    readonly="1"
+                    force_save="1"
+                    invisible="1"
+                />
```

Comment:
Why the 2 dates fields "use_date" and "expiration_date" are invisible ? As a consequence, when you are doing an inventory in manual mode, you don't see the expiration date of the lot that you are about to create.
I show the date field in my module stock_barcodes_gs1_expiry_simple (cf PR https://github.com/OCA/stock-logistics-barcode/pull/647) and it seems to work well. Is it a technical reason ?